package-lock.json

```diff
@@ -3204,7 +3403,7 @@
         "debug": "^4.1.1",
         "moment": "^2.24.0",
         "slugify": "^1.3.4",
-        "substitute": "https://github.com/segmentio/substitute/archive/0.1.0.tar.gz"
+        "substitute": "https://registry.npmjs.org/substitute/-/substitute-0.1.0.tgz"
```

Comment:
This isn't the same package and will revert when the metalsmith package that depends on it updates

Reply:
Good catch, thx! And since that `substitute` package doesn't seem like it has ever been published to npm, it looks like we don't have much choice but to allow packages from github.com afterall 😞

Reply:
Yeah. Feel free to make an upstream issue for this :)

Reply:
Opened https://github.com/segmentio/metalsmith-permalinks/issues/109